mechanisms/stochastic/noisy_expsyn_curr.mod

```diff
@@ -0,0 +1,87 @@
+: Exponential current-based synapse
+: Input current modeled by an Ornstein-Uhlenbeck process
```

Comment:
```suggestion
: Ornstein-Uhlenbeck input
: Can be used to model the input current from a population of exponential current-based synapses.
```
I'm suggesting this because from a neuroscience perspective, it is not one single synapse being modeled here, but a population. Accordingly, I think the filename should also be changed, e.g. to `ornstein_uhlenbeck.mod`.

Reply:
ok, thanks, that makes sense. However, Ornstein-Uhlenbeck is a particular mathematical equation which can be used to model many things. Can we come up with a more descriptive name which reflects the neuroscience?

Reply:
What about `ornstein_uhlenbeck_input` or `ou_input`? I guess that would account for the neuroscience aspect of it, modeling synaptic input.